test/config.yaml

```diff
@@ -0,0 +1,29 @@
+### general ###
+name: zipper-chn
```

Comment:
I standardized the naming convention for local name and downstreams name. @wujunzhuo